src/Appwrite/Certificates/Adapter.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Appwrite\Certificates;
+
+use Utopia\Logger\Log;
+
+interface Adapter
```

Comment:
Not clear why we need both adapter class and adapter provider class, seems over complex.